src/CreateCategoryModal.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+    Box,
+    Text,
+    NumberInput,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    useDisclosure,
+    Button,
+    Input,
+    Stack,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper
+  } from "@chakra-ui/react"
+import * as React from "react"
+
+interface CategoryModal {
+  goals: string[];
+}
+/*
+void()=> print(onCloseFunction:Function) {
+  console.log("test");
+  onCloseFunction();
+}
+*/
+export const CreateCategoryModal = (props: CategoryModal) => {
+    const{isOpen, onOpen, onClose} = useDisclosure()
+    
+    
+
+    return (
```

Comment:
We should use a style formatter so that our files get saved automatically and they look consistent across the codebase. We can discuss this more in detail but I was using the `prettier` extension for mine, and can decide on one to use